README.md

```diff
@@ -40,6 +40,19 @@ We have chosen the Rust programming language for Bee because of its C/C++ like p
 
 :::
 
+## Development
+
+The Bee repository has different branches:
+
+|Branch|Description|
+|------|-----------|
+|`mainnet`|The latest release for the IOTA protocol|
```

Comment:
```suggestion
|`mainnet`|The latest release for the IOTA network|
```

Reply:
Yeah, we have things mixed up. Each branch actually contains many _networks_ (each with their respective `config.*.json`). Shimmer will also have a `mainnet`, `testnet`, and so on. I think `mainnet` is the wrong name here, it should probably be just `iota`.